canonical/cloud-init
# Integration test for pull #586

## Proposed Commit Message
Integration test for pull #586

If a non-default AuthorizedKeysFile is specified in
/etc/ssh/sshd_config,  ensure we can still ssh as expected

## Additional Context
See https://github.com/canonical/cloud-init/pull/586

## Test Steps
Until SRU ends...
Run `pytest -s tests/integration_tests/bugs/test_gh586.py` to reproduce issue.
Run `CLOUD_INIT_CLOUD_INIT_SOURCE=PROPOSED pytest -s tests/integration_tests/bugs/test_gh586.py` to verify.

## Checklist:
